parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -231,6 +241,42 @@ private WriteBuilder createContextFunc(
       return this;
     }
 
+    private void setBloomFilterConfig(
+        Context context, BiConsumer<String, Boolean> withBloomFilterEnabled) {
+      Splitter dotSplitter = Splitter.on('.');
+      Joiner dotJoiner = Joiner.on(".");
+      context
+          .columnBloomFilterEnabled()
+          .forEach(
+              (columnPath, value) -> {
+                // check the column exists and support bloom filter
+                Types.NestedField field = schema.caseInsensitiveFindField(columnPath);
+                if (field == null) {
+                  LOG.warn(
+                      "Column: {} is not found in the schema: {}, ignore it.", columnPath, schema);
+                  return;
+                }
+                Type fieldType = field.type();
+                // doesn't support boolean, list, map, struct type
+                if (!fieldType.isPrimitiveType()
+                    || fieldType.typeId().equals(Type.TypeID.BOOLEAN)) {
+                  LOG.warn(
+                      "The type: {} of column: {} is not supported for parquet bloom filter, ignore it",
+                      fieldType,
+                      columnPath);
+                  return;
```

Comment:
Nit / non-blocking question: Is this something we can prevent the user from doing? That might not be worth it though.

Id also format it as `Invalid bloom filter column configuration: field `%s` is of type '%s` which parquet does not have bloom filter support for. This column configuration will be ignored.`. I am a fan of a full constant string at the front for searching logs.

Reply:
Thanks @kbendick for reviewing.

> Is this something we can prevent the user from doing? That might not be worth it though.

I think we may add some checks for adding/updating table properties.

> Id also format it as Invalid bloom filter column configuration: field %s is of type '%s' which cannot benefit from a bloom filter. This column configuration will be ignored.. I am a fan of a full constant string at the front for searching logs.

Updated.